### PR TITLE
Smoke test updates after 63

### PR DESCRIPTION
**Pull request by DanielMurphy22**
Addresses the points in the first comment here: https://github.com/mantidproject/mantid/issues/33519

The new issue creation has been tested here: https://github.com/DanielMurphy22/SmokeTests/issues/976

### CODE REVIEW

**Review by martyngigg — CHANGES_REQUESTED** [review]
Thanks for the updates.

I think `.idea/workspace.xml` might have snuck in from PyCharm?

**Comment by DanielMurphy22** [comment]
> Thanks for the updates.
> 
> I think `.idea/workspace.xml` might have snuck in from PyCharm?

Yes! I had noticed and was just messaging Conor about this! I want to dry run this on my personal repo first, but it'd be really good if you have a scan over the create_smoke_issues_OS.py script at some stage!

**Comment by DanielMurphy22** [comment]
I think the script create_smoke_issues_OS.py works now, but I still hit the secondary rate limits despite having the second waits: https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-secondary-rate-limits

File: Project-Management/Tools/RoadmapUpdate/SmokeTesting/create_smoke_issues_OS.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+- Comment on the testing issue with any issues you came across.
+- If no major issues were found, please **close the testing issue**.
+
+If you have any questions please contact the creator of this issue.
+
+### Tick below before testing the related issue
+
+'''
+
+def main() -> int:
```
Comment (martyngigg) [inline Project-Management/Tools/RoadmapUpdate/SmokeTesting/create_smoke_issues_OS.py:51]: Nice to see some type information!

**Review by martyngigg — CHANGES_REQUESTED** [review]
I really like to new smoke testing script. Just one question about it.

**Review by martyngigg — APPROVED** [review]

**Pull request merged**